src/Envelopes/Builder.php

```diff
@@ -2144,6 +2345,7 @@ public function addBrowserData(
         $languageSystem = '',
         $cookieEnabled = null,
         $doNotTrack = null,
+        $anonymous = null,
```

Comment:
Lets add to the end of params list for backward compatibility. Check all public methods.